src/_includes/base.njk

```diff
@@ -5,10 +5,11 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="{{ '/css/style.css' | url }}" />
+    <script src="{{ '/js/index.js' | url }}"></script>
     <title>{{title}}</title>
   </head>
   <body>
-    <div><a href="https://pittcsc.org/" class="branch-link">A Pitt CSC Branch 🌿</a></div>
+    <div><a href="https://pittcsc.org/" class="branch-link">A Pitt CSC Website 🌿</a></div>
```

Comment:
Think we should get rid of the branch emoji if we change it from "Branch" to "Website"?

Reply:
I was too stupid to get the pun, we can definitely keep "branch"